### PR TITLE
2018-07-09-team-sectumsempra.md

### PR DESCRIPTION
**Pull request by avneetreen**
Team Sectumsempra blog.
RGSoC 2018

### CODE REVIEW

File: blog/_posts/Hola, Amigos.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
+# "We solemnly swear we’re up to no good."
+
```
Comment (v-thakkar) [inline blog/_posts/Hola, Amigos.md]: Hi @avneet14027  ! Thanks for the PR. Please check the other blog posts for the proper markdown format. We need the whole set of parameters at the start.

File: blog/_posts/Hola, Amigos.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+ > *"I've always wanted to use that spell." ― Minerva McGonagall, Harry Potter and the Deathly Hallows*
+ 
+ 
+ 
+ ![Alt Text](https://assets.teenvogue.com/photos/590a3dda783fc429b031c296/master/w_757,c_limit/minerva%2520spel.gif)
+
+
+#### Looking forward to an funfilled summer,
+#### Team Sectumsempra
+
```
Comment (v-thakkar) [inline blog/_posts/Hola, Amigos.md]: No need of extra blank lines. Please remove them.

File: blog/_posts/Hola, Amigos.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+<pre> 
+        “Lives of great men, all remind us,
+        That we can make our lives sublime,
+        And departing, leave behind, us, 
+        Footprints, on the sands of time” 
+                    - H.W. Longfellow </pre>
+ Footprints, is what we wish to leave behind. :D 
+ 
+ 
+ ![Alt Text](https://github.com/avneet14027/Miscellaneous/blob/master/WhatsApp%20Image%202018-07-09%20at%205.13.08%20PM.jpeg)
```
Comment (v-thakkar) [inline blog/_posts/Hola, Amigos.md]: Add image description and image-credits class for adding image credits. Same for the GIF at the bottom.

File: blog/_posts/Hola, Amigos.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
+# "We solemnly swear we’re up to no good."
+
+## How delayed rewards brought us extra happiness!
+After a thorough application process, a long wait and an interview, we thought we might just be able to make it through, and were eagerly waiting for that one “Welcome to RGSoC” email. In time, an email popped up in our feed which, lured us into thinking we emerged victorious,  but Alas! When the first round of scholars was announced, we were heart-broken that we didn't make the cut. However, the same email also stated that we were waitlisted candidates, giving us a glimmer of hope. With our fingers crossed, we wished for the best! Almost after another month long wait, we were no less than flabbergasted, when we saw an email from RGSoC, quietly sitting in our inbox, giving us the much awaited news - We were officially in! And we couldn’t contain our excitement thereafter!
```
Comment (v-thakkar) [inline blog/_posts/Hola, Amigos.md]: Change victorious -> victoriously.

File: blog/_posts/Hola, Amigos.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+
+I am also a sports enthusiast, with basketball being one of my favourite sports. A book in hand, headphones and a decent internet connection is all I need when in melancholy. Quora comes to the rescue when the critic in me tends to rise.
+
+Elevating Medical Sciences using computers and technology is my passion, and it will be my future. I want to become a Computer Scientist and be known for my talent, and not because I’m a *woman*. I want people to understand that *I am more than what they can see*."
+
+I have had this habit of imaginng things in my head, things that I aspire to do and achieve, but the results are not always the way I expected them to be. But then, I take a deep breath and reflect back, and am reminded of the famous quote from the *Harry Potter series*
```
Comment (v-thakkar) [inline blog/_posts/Hola, Amigos.md]: Fix the typo here: 'imagining'.

File: blog/_posts/Hola, Amigos.md [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+
+### Our mentor:
+[**Michael Ferguson**](https://github.com/mppf) and [**Ben Albrecht**](https://github.com/ben-albrecht):
+
+We are delighted to have them as our mentors for the project, and hope that under their able guidance, we can have a steep learning curve. Also, we are grateful to them for helping us during the application process.
+
+### Our Official coaches:
+We are grateful to them for agreeing to coach us, guide us, and help us grow and learn, as we embark upon this journey. Again, we are extremely thankful to him for all the mentoring at the time of applications.
+
+ - [**Louis Jenkins**](https://louisjenkinscs.github.io/): He is a  recent Computer Science graduate from Bloomsburg interests and experience in the low-level areas such as compilers, operating systems, and the design and implementation of high-performance data structures. [Source](https://louisjenkinscs.github.io/)
```
Comment (v-thakkar) [inline blog/_posts/Hola, Amigos.md]: Not sure why you have added the same website/linkedin links twice. Please remove the links with 'Source' texts.

File: blog/_posts/Hola, Amigos.md [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+- [**Sarthak Munshi**](https://saru.science/about/): He is currently, working at HackerRank. Most of his work is open-sourced and can be found on his [github profile](https://github.com/saru95). [Source](https://saru.science/about/)
+
+- [**Engin Kayraklioglu**](https://github.com/e-kayrakli)
+
+### Volunteer Coaches: 
+They are willing to help us at any point of time during the project and we are thankful to them for that. 
+
+- [**Kushal Singh**](https://in.linkedin.com/in/kushalsingh007): He is currently working as a platform engineer at Direct-I, India, and has previously been part of Google Summer Of Code. [Source](https://in.linkedin.com/in/kushalsingh007)
+
+- [**Lydia Duncan**](https://github.com/lydia-duncan)
```
Comment (v-thakkar) [inline blog/_posts/Hola, Amigos.md]: It looks weird to have small introduction text for everyone except this person. Please either add for all or none.

**Review by v-thakkar — CHANGES_REQUESTED** [review]

**Comment by avneetreen** [comment]
Hi Vaishali,
I have made the changes as suggested by you. 
Thanks,
Avneet

**Review by v-thakkar — APPROVED** [review]

File: blog/_posts/Hola, Amigos.md [hunk 8]
```diff
@@ -0,0 +1,97 @@
+---
+**title:** We solemnly swear we’re upto no good!
```
Comment (v-thakkar) [inline blog/_posts/Hola, Amigos.md]: Hi @avneet14027 ! You would need to use exact format as used in other files for declaring parameters. For example, no use of '**', permalink same as file name etc. Please check and test them locally.

File: blog/_posts/Hola, Amigos.md [hunk 9]
```diff
@@ -0,0 +1,97 @@
...
+And hence began our official journey with RGSoC, and while currently working on our first task, all we wish for and would work towards is making an impact, and let our work speak for ourselves. We are reminded of a few lines, which we truly resonate with: 
+ 
+>        “Lives of great men, all remind us,
+>        That we can make our lives sublime,
+>        And departing, leave behind, us, 
+>        Footprints, on the sands of time”  
+>                    - H.W. Longfellow 
+ Footprints, is what we wish to leave behind. :D 
+ 
+ ![Alt Text](https://github.com/avneet14027/images/blob/master/2018-07-09-team-sectumsempra-blog-teampic.jpeg)
```
Comment (v-thakkar) [inline blog/_posts/Hola, Amigos.md]: Please add image description as well. For example, who is in the image etc. And then in brackets, you can write about image source. As the blog post is from a team and not a single person, 'self' is a confusing word as a source. You need to specify the name or device etc.

**Review by v-thakkar — CHANGES_REQUESTED** [review]

**Comment by avneetreen** [comment]
Hey @nerdyvaishali , I have updated the file as per the changes requested!
Thanks,
Avneet

**Review by v-thakkar — APPROVED** [review]

**Pull request merged**